arangod/Transaction/Methods.cpp

```diff
@@ -1962,23 +2010,23 @@ OperationResult transaction::Methods::countLocal(std::string const& collectionNa
   DataSourceId cid = addCollectionAtRuntime(collectionName, AccessMode::Type::READ);
   auto const& collection = trxCollection(cid)->collection();
 
-//  Result lockResult = lockRecursive(cid, AccessMode::Type::READ);
-//
-//  if (!lockResult.ok() && !lockResult.is(TRI_ERROR_LOCKED)) {
-//    return OperationResult(lockResult);
-//  }
+  //  Result lockResult = lockRecursive(cid, AccessMode::Type::READ);
+  //
+  //  if (!lockResult.ok() && !lockResult.is(TRI_ERROR_LOCKED)) {
+  //    return OperationResult(lockResult);
+  //  }
```

Comment:
```suggestion
  //  Result lockResult = lockRecursive(cid, AccessMode::Type::READ);
  //
  //  if (!lockResult.ok() && !lockResult.is(TRI_ERROR_LOCKED)) {
  //    return OperationResult(lockResult);
  //  }
```

Can just simply be removed?

Reply:
Yes please remove...